config/application.yml.example

```diff
@@ -10,7 +10,7 @@
 
 development:
   allow_third_party_auth: 'yes'
-  domain_name: 'xxx.xxx'
+  domain_name: 'localhost:3000'
   ducksboard_uid: '8VEQre2c8b62mQV0sa'
   idp_sso_target_url: 'https://upaya-mockidp.18f.gov/saml/auth'
```

Comment:
I setup https://upaya-idp-dev.18f.gov/ and https://upaya-idp.18f.gov/ if we reuse one of those